4회 4주차/최대공약수와 최소공배수.js

```diff
@@ -0,0 +1,12 @@
+function solution(n, m) {
+    const gcd = (a, b) => {
+        while (b !== 0) {
+            let temp = b;
+            b = a % b;
+            a = temp;
+        }
+        return a;
+    };
```

Comment:
```suggestion
 const gcd = (a, b) => b ? gcd(b, a % b) : a;
```
유클리드 호제법을 재귀로 간결하게 표현할 수 있을것 같아용 !!